### PR TITLE
[IMP] agreement_serviceprofile

### PR DESCRIPTION
**Pull request by max3903**

### CODE REVIEW

**Comment by max3903** [comment]
Ping @barnarj @kvedder-amplex

File: agreement_serviceprofile/models/agreement_serviceprofile.py [hunk 1]
```diff
@@ -26,7 +26,10 @@ def _default_stage_id(self):
...
     notes = fields.Text(string="Notes")
     product_id = fields.Many2one('product.template', 'Service',
-                                 domain="[('type', '=', 'service')]")
+                                 domain="[('type', '=', 'service')]",
+                                 required=True)
+    partner_id = fields.Many2one(related='agreement_id.partner_id',
+                                 string='Partner')
 
     # Used for Kanban grouped_by view
     @api.model
```
Comment (kvedder-amplex) [inline agreement_serviceprofile/models/agreement_serviceprofile.py:35]: These changes should be helpful, thanks!

File: agreement_serviceprofile/models/agreement_serviceprofile.py [hunk 2]
```diff
@@ -26,7 +26,10 @@ def _default_stage_id(self):
 
     notes = fields.Text(string="Notes")
     product_id = fields.Many2one('product.template', 'Service',
```
Comment (patrickrwilson) [inline agreement_serviceprofile/models/agreement_serviceprofile.py]: Might make more sense for the string to be 'Service Product'

**Review by patrickrwilson — APPROVED** [review]
Just one small suggestion. Looks good.

**Review by bodedra — APPROVED** [review]
LGTM :+1:

**Pull request merged**